spec/controllers/bills_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+describe BillsController do
+
+  describe "GET #index" do
+    let!(:bill) { FactoryGirl.create(:bill) }
```

Comment:
This line is repeated a few times here, any advice on how to dry it up would be great!